app/models.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser
+from django.contrib.auth.models import PermissionsMixin
+from django.utils.translation import gettext_lazy as _
+from django.utils import timezone
+
+from .managers import CustomUserManager
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+    email = models.EmailField(_('email address'), unique=True)
+    username = models.CharField(max_length=50)
+    date_of_birth = models.DateField(null=True)
+    phone_no = models.CharField(max_length=10)
+    willin_to_help = models.IntegerField(default=1)     # 1-Willing to help, 0-Not Willing to help
+    is_staff = models.BooleanField(default=False)       # Field necessary for a django user
+    is_active = models.BooleanField(default=True)       # Field necessary for a django user
+    is_superuser = models.BooleanField(default=False)   # Field necessary for a django user
+    date_joined = models.DateTimeField(default=timezone.now)
+
+    USERNAME_FIELD = 'email'
+    REQUIRED_FIELDS = ['username', 'date_of_birth', 'phone_no', 'device_id']
+
+    objects = CustomUserManager()
+
+    def __str__(self):
+        return self.email
+
+
+class Requests(models.Model):
```

Comment:
rename this model to connection request something, Ambiguity will come because its similar to requests

Reply:
Ok